content/en/opentelemetry/config/environment_variable_support.md

```diff
@@ -29,21 +29,36 @@ further_reading:
       text: 'Ruby SDK Configuration'
 ---
 
-Datadog SDKs implement the OpenTelemetry Tracing APIs, allowing you to use OpenTelemetry environment variables to configure Datadog tracing for your applications. Replace the OpenTelemetry SDK with the Datadog SDK in your application to receive traces and additional Datadog telemetry with minimal changes to your existing configuration.
-This page describes the OpenTelemetry SDK options Datadog supports.
+Datadog SDKs implement the OpenTelemetry APIs for traces, metrics, and logs. This allows you to use standard OpenTelemetry environment variables to configure your application while receiving additional Datadog telemetry with minimal changes to your existing configuration.
+
+This page describes the environment variables Datadog supports for OpenTelemetry interoperability.
 
 <div class="alert alert-info">If both Datadog and OpenTelemetry environment variables are set, Datadog takes precedence. Datadog defaults also override OpenTelemetry defaults. See the relevant <a href="/tracing/trace_collection/library_config/">SDK Configuration page</a> for default values and more information.</div>
 
+## Datadog SDK configuration
+These environment variables enable the Datadog SDK to ingest OpenTelemetry Metrics and Logs API data. For guides on how to instrument your application, see the [language-specific instrumentation documentation][14].
+
+`DD_METRICS_OTEL_ENABLED`
+: **Description**: Enables the automatic instrumentation components of the Datadog SDK to collect and emit metrics generated by the OTel Metrics API. <br>
+**Default**: `false`
+
+`DD_LOGS_OTEL_ENABLED`
+: **Description**: Enables the automatic instrumentation components of the Datadog SDK to collect and emit logs generated by the OTel Logs API. <br>
+**Default**: `false`
+
 ## General SDK configuration
 Datadog SDKs support the following general OpenTelemetry SDK options. For more information, see the related [OpenTelemetry documentation][9].
 
-`OTEL_SERVICE_NAME`
-: ****Datadog convention****: `DD_SERVICE`<br>
-Sets the service name<br>
-**Notes**: If `service.name` is also provided in `OTEL_RESOURCE_ATTRIBUTES`, then `OTEL_SERVICE_NAME` takes precedence<br>
+`service.name` (Resource Attribute)
```

Comment:
I think we should revert to the previous description while also adding the `service.name` resource attribute info to the `OTEL_RESOURCE_ATTRIBUTES` description later.

> This is resolved as a resource attribute, not from `OTEL_SERVICE_NAME`.
I'm not sure where this info came from, but `OTEL_SERVICE_NAME` [is responsible](https://opentelemetry.io/docs/specs/otel/configuration/sdk-environment-variables/#general-sdk-configuration) for setting the `service.name` resource attribute. It's a little weird because in our traces we don't really use resource attributes, but in our metrics and logs we do.